src/dashboard/install/README.md

```diff
@@ -150,28 +206,79 @@ This is the full list of environment strings supported:
     - **Type:** `float`
     - **Default:** `0`
 
+### Gunicorn variables
 
-## Other configuration files
+- **`AM_GUNICORN_USER`**:
+    - **Description:** OS user for gunicorn worker processes to run as.  See [USER](http://docs.gunicorn.org/en/stable/settings.html#user)
```

Comment:
All the _Description_ fields under _Gunicorn variables_ are missing the period at the end of the sentence.